static/js/message_list.js

```diff
@@ -287,12 +287,14 @@ export class MessageList {
         }
         $row.find(".message_edit_form").append(edit_obj.$form);
         $row.find(".message_content, .status-message, .message_controls").hide();
+        $row.find(".sender-status").css("margin", "0 3px 0 0");
         $row.find(".message_edit").css("display", "block");
         autosize($row.find(".message_edit_content"));
     }
 
     hide_edit_message($row) {
         $row.find(".message_content, .status-message, .message_controls").show();
+        $row.find(".sender-status").css("margin", "8px 0 8px -3px");
```

Comment:
We try to avoid making direct CSS changes in JavaScript; preferring to make any changes via toggling CSS classes.

Reply:
@timabbott  I have updated the PR you can re-review.